### ISSUE TITLE
unable to prevent some classes from being scoped.

### ISSUE THREAD

**Issue by TheMcMurder** [CanopyTax/kremling-loader#10]
example:

````html
<div class='myGlobalClass'>
  <MyComponent />
</div>
````

````jsx
import css from './css'
function MyComponent() {
  const scope = useCss(css)
 return (
   <div {...scope} className='myScopedClass'>
    Hello   
   </div>
 )
}
````

````css
.myGlobalClass .myScopedClass {
  color: red;
}
````

I'd like a way to signal to the kremling loader that `myGlobalClass` shouldn't be scoped. An example of this using CSS modules looks like this.

````css
:global(.myGlobalClass) .myScopedClass {
  color: red;
}
````

As far as I can tell currently the kremling loader doesn't support _not_ scoping a portion of the css selector.

**Comment by geoctrl** [comment]
Correct, kremling-loader doesn't support global styling... 

There's a couple ways to solve this

You can add inline styling for your global styles:

```javascript
const css = `
  .myGlobalClass {
    // styles
  }
`;
```

Or you can add another webpack rule that doesn't run css through the kremling-loader (maybe targeting files with a different naming match - eg: `*.global.css`)

**Comment by joeldenning** [comment]
Yeah if you're not using kremling-loader this is possible with what @geoctrl said. We could transfer this issue to kremling-loader's repo, maybe, if we want to support that within kremling-loader?

**Comment by geoctrl** [comment]
fixed in #14
